index.js

```diff
@@ -65,49 +60,53 @@ module.exports = async (target, options) => {
 			cliArguments.push('--background');
 		}
 
-		if (options.app) {
-			cliArguments.push('-a', options.app);
+		if (app) {
+			cliArguments.push('-a', app);
 		}
 	} else if (process.platform === 'win32' || (isWsl && !isDocker())) {
-		command = 'cmd' + (isWsl ? '.exe' : '');
-		cliArguments.push('/s', '/c', 'start', '""', '/b');
+		command = 'powershell' + (isWsl ? '.exe' : '');
```

Comment:
Is it safe to use Powershell for the non-WSL case too? Do all Windows installations have it?

Reply:
Yeah it's comes with every windows installation, it's the defacto interpreter for the .NET ecosystem, so it's not at all related to wsl and will work for everyone. I also tested it on both wsl and in windows–using powershell 😅